gapic/templates/%namespace/%name_%version/%sub/services/%service/_shared_macros.j2

```diff
@@ -383,13 +383,32 @@ class {{ async_method_name_prefix }}{{ service.name }}RestInterceptor:
     {% endif %}
         """Post-rpc interceptor for {{ method.name|snake_case }}
 
-        Override in a subclass to manipulate the response
+        Deprecated. This interceptor is deprecated. Please use the `post_{{ method.name|snake_case }}_with_metadata`
+        interceptor instead. Override in a subclass to read or manipulate the response
         after it is returned by the {{ service.name }} server but before
-        it is returned to user code.
+        it is returned to user code. This `post_{{ method.name|snake_case }}` iterceptor runs
+        before the `post_{{ method.name|snake_case }}_with_metadata` iterceptor.
         """
```

Comment:
I suggest

```
        DEPRECATED. Please use the `post_{{ method.name|snake_case }}_with_metadata`
        interceptor instead. 

        Override in a subclass to read or manipulate the response
        after it is returned by the {{ service.name }} server but before
        it is returned to user code. This `post_{{ method.name|snake_case }}` interceptor runs
        before the `post_{{ method.name|snake_case }}_with_metadata` interceptor.
```

(reworded slightly and fixed spelling of "interceptor")

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2299/commits/83cfa50c7cb14f663f540d270e7d85c8f5559b85. Good catch on the spelling error. I filed https://github.com/googleapis/gapic-generator-python/issues/2301 to see if we can add a presubmit to help with future development.